wiseService/wiseService.js

```diff
@@ -847,12 +846,12 @@ app.get('/stats', [noCacheJson], function (req, res) {
   res.send(stats);
 });
 // ----------------------------------------------------------------------------
-if (getConfig('wiseService', 'regressionTests')) {
-  app.post('/shutdown', (req, res) => {
+app.post('/shutdown', (req, res) => {
```

Comment:
Why the change? I think not having api available is better.

Reply:
👍